index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Newton Portfolio Project</title>
+  <link rel="stylesheet" href="/style.css">
+</head>
+<body>
+  <main>
+    <nav>
+      <div class="desktop-menu">
+        <ul>
+          <li>
+            <a href="">Portfolio</a>
+          </li>
+          <li>
+            <a href="">About</a>
+          </li>
+          <li>
+            <a href="">Contact</a>
+          </li>
+        </ul>
+      </div>
+      <div class="mobile-menu-icon">
+        <img src="./images/Union.svg" alt="">
+      </div>
```

Comment:
### Toolbar
![image](https://github.com/NewtonOmbese/Newton-Portfolio/assets/78251927/8e3618c1-5876-4c05-b2ea-c17c6523ffbb)

It would be nice if we wrap the entire section in a `header` tag. 

- Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. 👍

Reply:
@abdulhamiid  I did this
![Screenshot from 2023-06-01 08-58-35](https://github.com/NewtonOmbese/Newton-Portfolio/assets/79839603/7d4809b4-6597-4a8e-88de-7e37c8db010e)